source/topic5.rst

```diff
@@ -7,8 +7,50 @@ Stack Operations
 ================
 
 
-Example Uses
-============
+Example Use
+===========
+
+* We'll use a stack to solve a problem without having to know how it is implemented
+    * We can even write the pseudocode for an algorithm using a stack
+
+Postfix Expressions
+-------------------
+
+* We as humans prefer *infix* notation for our mathematical expressions
+    * 3 + 4 * 2 = 11
+    * (7 + 2) / 3 = 3
+
+* But this is just a convention
+* We can use other notations, like *postfix* notation
+    * 3 4 2 * + = 11
+    * 7 2 + 3 / = 3
```

Comment:
Postfix is extremely uncommon. I'd clarify this with some backwards-lisp-esque expansion:
* `3 4 2 * + = (3 (4 2 *) +) = 11`
* `7 2 + 3 /  = ((7 2 +) 3 /) = 3`

May also help illustrate why the algorithm below works before they dig into the larger example.

Reply:
Defo will add this.